examples/MerminMagic/main.jl

```diff
@@ -0,0 +1,59 @@
+# Mermin-Peres Magic Square Game Algorithm
+# https://www.gregegan.net/SCIENCE/MPMS/MPMS.html
+using Yao
+using Yao.EasyBuild, YaoPlots
+using Random
+
+# initialize four qubits to be two pairs of Bell Pairs (00)
+bell_state_00(ctr_idx::Int) = chain(4, put(ctr_idx => H), control(ctr_idx, (ctr_idx + 1) => X))
```

Comment:
It should be a two qubit gate. If you want to apply this two qubit gate on any location, just use `put`
```julia
julia> using Yao

julia> reg = rand_state(4);

julia> apply!(reg, put(4, (2, 1)=>kron(X, X)))
ArrayReg{2, ComplexF64, Array...}
    active qubits: 4/4
    nlevel: 2
```